### DISCUSSION TITLE
cannot connect to postgres on nhost.io, but dbeaver is OK

### DISCUSSION THREAD

**Discussion by guotie** [prisma/prisma#20749] [Q&A]
### Bug description

I set up a postgres instance on nhost.io, when I run 
```
npx prisma migrate dev --name init
```

the following error occurs:
```
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": PostgreSQL database "xifevrvevugydklyopgq", schema "public" at "xifevrvevugydklyopgq.db.ap-southeast-1.nhost.run:5432"

Error: db error: FATAL: password authentication failed
   0: schema_core::state::DevDiagnostic
             at schema-engine/core/src/state.rs:270
```

But I use the same host, dbname, password, dbeaver connect is OK

my .env:
```
# DATABASE_URL="postgresql://postgres:xxxxx@xifevrvevugydklyopgq.db.ap-southeast-1.nhost.run:5432/xifevrvevugydklyopgq?schema=public"
```

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```
Environment variables loaded from .env
prisma                  : 5.1.1
@prisma/client          : 5.1.1
Current platform        : darwin-arm64
Query Engine (Node-API) : libquery-engine 6a3747c37ff169c90047725a05a6ef02e32ac97e (at ../../node_modules/.pnpm/@prisma+engines@5.1.1/node_modules/@prisma/engines/libquery_engine-darwin-arm64.dylib.node)
Schema Engine           : schema-engine-cli 6a3747c37ff169c90047725a05a6ef02e32ac97e (at ../../node_modules/.pnpm/@prisma+engines@5.1.1/node_modules/@prisma/engines/schema-engine-darwin-arm64)
Schema Wasm             : @prisma/prisma-schema-wasm 5.1.1-1.6a3747c37ff169c90047725a05a6ef02e32ac97e
Default Engines Hash    : 6a3747c37ff169c90047725a05a6ef02e32ac97e
Studio                  : 0.492.0
```

**Comment by guotie** [comment]
but db push works.

**Comment by ludralph** [comment]
Hi @guotie 👋 

Can you please create a [shadow database](https://www.prisma.io/docs/concepts/components/prisma-migrate/shadow-database#cloud-hosted-shadow-databases-must-be-created-manually) as suggested in this open issue?

- https://github.com/prisma/prisma/issues/16170#issuecomment-1362547341


If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
